src/Tribe/Repository.php

```diff
@@ -606,13 +606,15 @@ public function found() {
 		$query = $this->build_query();
 
 		// The request property will be set during the `get_posts` method and empty before it.
-		if ( ! empty( $query->request ) ) {
+		if ( ! empty( $query->request ) && ( 0 < $query->found_posts || ! $this->skip_found_rows ) ) {
```

Comment:
Probably personal preference but I think using `! $query->found_posts` is more expressive than `0 < $query->found_posts` and provides the same result as `0` will evaluate to `false`

Reply:
✅